tests/Tests/400_ods_dashboard/400_ods_dashboard.robot

```diff
@@ -42,4 +49,49 @@ Dashboard Test Setup
   Wait for RHODS Dashboard to Load
 
 Dashboard Test Teardown
-  Close All Browsers
\ No newline at end of file
+  Close All Browsers
+  
+Get HTTP Status Code
+    [Arguments]  ${link_to_check}
+    ${response}=    RequestsLibrary.GET  ${link_to_check}   expected_status=any
+    Run Keyword And Continue On Failure  Status Should Be  200
+    [Return]  ${response.status_code}
+
+Check All Links Are working
+  [Documentation]  Verify the all links after appling a filter
+  ${link_elements}=  Get WebElements    //a[@class="odh-card__footer__link" and not(starts-with(@href, '#'))]
+  ${len}=  Get Length    ${link_elements}
+  Log To Console    ${len} Links found\n
+  FOR  ${idx}  ${ext_link}  IN ENUMERATE  @{link_elements}  start=1
+      ${href}=  Get Element Attribute    ${ext_link}    href
+      ${status}=  Get HTTP Status Code   ${href}
+      Log To Console    ${idx}. ${href} gets status code ${status}
+  END
+
+Check For enabled option
+  [Documentation]  Filter Resources by enabling "enabled" from enable state
+  ${link_elements}=  Get WebElements    //a[@class="odh-card__footer__link"]
+  ${len}=  Get Length    ${link_elements}
+  Select Checkbox  id=enabled-filter-checkbox--check-box
+  sleep  10s
+  ${link_elements}=  Get WebElements    //a[@class="odh-card__footer__link"]
+  ${lent}=  Get Length    ${link_elements}
+  Should Not Be Equal As Integers  ${len}  ${lent}
```

Comment:
I think it may make sense to check also that after the filter has been applied, you get the expected items. So for example, if the only application enabled is Anaconda, I would expect to find only items about Anaconda when using this filter